packages/@coorpacademy-components/src/molecule/bulk-progress-bar/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import ProgressBar from '../progress-bar';
+import StatusItem from '../../atom/status-item';
+import style from './style.css';
+import {Props, propTypes} from './types';
+
+const STATUS = {
+  inProgress: 'inProgress',
+  fail: 'fail'
+};
+
+const renderStatusIcon = (status: string, progress: number) => {
+  if (status === STATUS.inProgress)
+    return <div className={style.progressText}>{`${progress} %`}</div>;
+  return <StatusItem icon={status === STATUS.fail ? 'invalid' : 'valid'} />;
+};
+
+const resolveProgressBarColor = (status: string) => {
```

Comment:
utilise le type Status importé de types (regarde en plus loin dans la review)
```suggestion
const resolveProgressBarColor = (status: Status) => {
```